### DISCUSSION TITLE
Hvordan kan jeg se vejrudsigten i codeimprover?

### DISCUSSION THREAD

**Discussion by star-kasperbirkelundlarsen** [adjursaaFoxGmail/lowq#8] [Q&A] [answered]
Hej Anders - kan du selv definere kategorierne?

**Comment by kasperbirkelund** [comment]
Hej fra en anden git konto

**Reply by adjursaaFoxGmail** [reply]
Det virker også :-)

**Comment by adjursaaFoxGmail** [comment, marked as answer]
> Hej Anders - kan du selv definere kategorierne?

Ja - det kan man.
Når man definerer en kategori så skal man afgøre om det er a typen diskussion - eller typen Q&A
Venlig hilsen Den venlige redaktør :-)

**Comment by adjursaaFoxGmail** [comment]
Vejrudsigten er der derimod lidt længere udsigter til at se
Venlig hilsen :-)
